birdhouse/components/cowbird/config/canarie-api/cowbird_config.py.template

```diff
@@ -0,0 +1,38 @@
+SERVICES['Cowbird'] = {
+    'info': {
+        'name': 'Cowbird',
+        'synopsis': 'Cowbird is a middleware that manages interactions between various birds of the bird-house stack.',
+        'version': "${COWBIRD_VERSION}",
+        'institution': 'Ouranos, CRIM',
+        'releaseTime': get_release_time_from_repo_tag("github", "Ouranosinc/cowbird", "${COWBIRD_VERSION}"),
+        'researchSubject': 'Any',
+        'supportEmail': '${SUPPORT_EMAIL}',
+        'category': 'Security',
+        'tags': ['Security', 'Management', 'Access', 'Policy Decision Point']
+    },
+    'stats': {
+        'method': '.*',
+        'route': "/cowbird/.*"
+    },
+    'redirect': {
+        'doc': 'https://pavics-cowbird.readthedocs.io/',
+        'releasenotes': 'https://github.com/Ouranosinc/cowbird//blob/master/CHANGES.rst',
+        'support': 'https://github.com/Ouranosinc/cowbird//issues',
+        'source': 'https://github.com/Ouranosinc/cowbird/',
+        'tryme': 'https://${PAVICS_FQDN_PUBLIC}/cowbird/',
+        'licence': 'https://github.com/Ouranosinc/cowbird//blob/${COWBIRD_VERSION}/LICENSE',
+        'provenance': 'https://github.com/Ouranosinc/cowbird/'
+    },
+    "monitoring": {
+        "Cowbird": {
+            'request': {
+                'url': 'http://cowbird:7000/'
```

Comment:
It has a protected endpoint on `/cowbird/`, but admin permissions are required since cowbird is only for synching permissions/services. We could however add  `/cowbird/` and  `/cowbird/version` with exact path match as permitted publicly in Magpie.